backend/resources/n_shot_examples_event_driven.py

```diff
@@ -1,6 +1,135 @@
 from resources.state_machine_descriptions import *
 
 n_shot_examples = {
+    "dishwasher_winter_2019": {
+        "system_description": dishwasher_winter_2019,
+        "system_name": "DishwasherOn",
+        "states_table": """```html<table border="1">
+        <tr><th>StateName</th></tr>
+        <tr><td>Open</td></tr>
+        <tr><td>Closed</td></tr>
+        <tr><td>Locked</td></tr>
+        <tr><td>Idle</td></tr>
+        <tr><td>Cleaning</td></tr>
+        <tr><td>Intake</td></tr>
+        <tr><td>Washing</td></tr>
+        <tr><td>Drain</td></tr>
+        <tr><td>Drying</td></tr>
+        <tr><td>Suspended</td></tr>
+        </table>```""",
+
+        "initial_state": "Idle",
+
+        "events_table": """```html<table border="1">
+        <tr><th>EventName</th></tr>
+        <tr><td>Turn dishwasher on</td></tr>
+        <tr><td>Toggle dishwasher program</td></tr>
+        <tr><td>Toggle dishwasher drying runtime</td></tr>
+        <tr><td>Start the Dishwasher</td></tr>
+        <tr><td>Dishwasher is washing</td></tr>
+        <tr><td>Dishwasher Waiting</td></tr>
+        <tr><td>Dishwasher Draining</td></tr>
+        <tr><td>Dishwasher Paused</td></tr>
+        <tr><td>Dishwasher Drying</td></tr>
+        <tr><td>Extend Dishwasher Drying Time</td></tr>
+        <tr><td>Stop Dishwasher Drying</td></tr>
+        <tr><td>Dishwasher resumed using the same settings</td></tr>
+        <tr><td>Open Dishwasher Door</td></tr>
+        <tr><td>Close Dishwasher Door</td></tr>
+        <tr><td>Lock the dishwasher Door</td></tr>
+        <tr><td>Unlock the dishwasher Door</td></tr>
+        </table>```""",
+
+        "state_inspected": "Idle",
+
+        "associated_events": """
+        Toggle dishwasher program,
+        Toggle dishwasher drying runtime,
+        Start the Dishwasher,
+        Close Dishwasher Door, 
+        Lock the dishwasher Door,
+        Dishwasher Waiting, """,
+
+        "event_inspected": "Turn dishwasher on",
+
+        "create_transitions": """```html<table border="1">
+        <tr><th>From State</th><th>To State</th><th>Event</th><th>Guard</th><th>Action</th></tr>
+        <tr><td \>Idle</td><td>Idle</td><td>Toggle dishwasher program & Diswasher Waiting</td><td[>]Program number >= 1]</td><td>Diswasher repetitions = Program Number</td></tr>
+        <tr><td>Idle</td><td>Idle</td><td>Toggle dishwasher drying time & Diswasher Waiting</td><td>NONE</td><td>if drying time equal to 20 then set it to 40 otherwise set it to 20</td></tr>
+        <tr><td>Idle</td><td>Cleaning</td><td>Start the diswasher</td><td>[Close dishwasher door]</td><td>Lock the dishwasher door and set the counter = 1</td></tr>
```

Comment:
When this example is being used at this stage in the SMF, the superstates haven't been identified yet. Since `Cleaning` is a superstate, we should decompose the transitions to `Cleaning` to all of its substates

```suggestion
        <tr><td>Idle</td><td>Intake</td><td>Start the diswasher</td><td>[Dishwasher door is closed]</td><td>Lock the dishwasher door and set the counter = 1</td></tr>
        <tr><td>Idle</td><td>Washing</td><td>Start the diswasher</td><td>[Dishwasher door is closed]</td><td>Lock the dishwasher door and set the counter = 1</td></tr>
        <tr><td>Idle</td><td>Drain</td><td>Start the diswasher</td><td>[Dishwasher door is closed]</td><td>Lock the dishwasher door and set the counter = 1</td></tr>
```